tensorboard/webapp/runs/views/runs_table/runs_group_menu_button_component.ng.html

```diff
@@ -66,4 +66,18 @@
     </span>
     <label>Regex</label>
   </button>
+  <button
+    mat-menu-item
+    role="menuitem"
+    *ngIf="showGroupByRegex"
+    [attr.aria-checked]="selectedGroupBy.key === GroupByKey.REGEX"
```

Comment:
This should be removed; multi-select radio button does not make sense and it does not make sense for `menuitem` to have `aria-checked` property.

Reply:
so then the test `renders a check icon and aria-checked for the current groupBy menu item` the last one is expected to be `null`

Reply:
Ack. We should be `querySelectorAll`ing `menuitemradio` instead in the test.

Reply:
updated with "menuitemradio" and remove `null` etc in the test check